native/dlasq5.go

```diff
@@ -0,0 +1,125 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+import "math"
+
+// Dlasq5 computes one dqds transform in ping-pong form.
+// i0 and n0 are zero-indexed.
+func (impl Implementation) Dlasq5(i0, n0 int, z []float64, pp int, tau, sigma, dmin, dmin1, dmin2, dn, dnm1, dnm2 float64) (i0Out, n0Out, ppOut int, tauOut, sigmaOut, dminOut, dmin1Out, dmin2Out, dnOut, dnm1Out, dnm2Out float64) {
+	// TODO(btracey): It seems like the listed outputs of this function are actually
+	// outputs (dmin, dn, etc.). Make them true outputs.
+	// The lapack function has inputs for ieee and eps, but Go requires ieee so
+	// these are unnecessary.
+	if n0-i0-1 <= 0 {
+		return
```

Comment:
Does your comment above mean that this should return "i0, n0, pp, tau, sigma, dmin, dmin1, dmin2, dn, dnm1, dnm2"?